gen_cfg_section.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import os
+import re
+from urllib.request import urlopen
+
+import yaml
+
+
+def parse_cli_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        'api_versions',
+        nargs='+',
+        help=(
+            'Docker API version(s) to support. ' +
+            'If exactly two versions are passed, ' +
+            'it will act as a range.'
+        )
+    )
+    parser.add_argument(
+        '--yaml-dir',
+        help='Save and load Swagger YAML files from this directory'
+    )
+    parser.add_argument(
+        '--add-comments',
```

Comment:
Not needed. The comments are meaningless if you get the api version from the docker socket.